openapi_tester/schema_tester.py

```diff
@@ -413,27 +457,28 @@ def _test_openapi_type_object(
                 ignore_case=ignore_case,
             )
 
-    def _test_openapi_type_array(
+    def test_openapi_array(
         self,
         schema_section: dict,
         data: dict,
         reference: str,
         case_tester: Optional[Callable[[str], None]],
         ignore_case: Optional[List[str]],
     ) -> None:
-        items = schema_section["items"]
-
-        error = ""
-        if items is None and data is not None:
-            error = "Mismatched content. Response list contains data when the schema is empty."
-        elif data is None and not self.is_nullable(schema_section):
-            error = NONE_ERROR.format(expected="list")
-        elif data is None:
-            return
-
-        if error:
+        items = schema_section.get("items", {})
```

Comment:
From the OAS docs: `Unlike JSON Schema, the items keyword is required in arrays`. 

```suggestion
        items = schema_section['items']
```

Reply:
Ok, I encountered a situation where it wasnt - maybe due to the schema converter hiccup.

Reply:
In one of the tests invoking the schema validation as well? If this isn't covered by `openapi-spec-validator`, I guess it's even more important to enforce it here.

Reply:
I've temporarily removed the "broken" schemas in b28eb59. I'll merge this now so we can limit the number of conflicts in new branches, but let's talk more about whether we should reimplement these schemas and/or fix them. The OAS spec seems pretty clear on this.